db/schema.rb

```diff
@@ -226,12 +226,20 @@
     t.index ["resource_id"], name: "index_translations_on_resource_id"
   end
 
+  create_table "user_counter_values", force: :cascade do |t|
+    t.string "user_id"
```

Comment:
should this be `t.integer`?

Reply:
potentially `user_counter_id` has the same issue

Reply:
Oh, good catch! Yes

Reply:
does this same thing apply to `user_counter_id`?